### PR TITLE
Add datastore settings to the Web UI

### PR DESCRIPTION
**Pull request by murrant**
image

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by SourceDoctor** [comment]
looks a little unsorted in Screenshot ;-)
Datastores should all be before or after other fields, so it's better to be seen which are available

**Comment by PipoCanaja** [comment]
@murrant Concerning the order, we could either rename the config option and keep the text as it is, or put the "Datastore" after the name in the text. I would be more for the 1st option.

**Comment by murrant** [comment]
There is no way to order them right now, order is based on the group name.

File: resources/lang/en/settings.php [hunk 1]
```diff
@@ -45,8 +45,12 @@
         ],
         'poller' => [
             'distributed' => 'Distributed Poller',
+            'graphite' => 'Datastore: Graphite',
+            'influxdb' => 'Datastore: InfluxDB',
+            'opentsdb' => 'Datastore: OpenTSDB',
             'ping' => 'Ping',
```
Comment (SourceDoctor) [inline resources/lang/en/settings.php]: @murrant Reordering here handles the Problem
It's a list..

Comment (murrant) [reply resources/lang/en/settings.php]: Nope: https://github.com/librenms/librenms/blob/master/resources/js/components/LibrenmsSettings.vue#L166

Comment (murrant) [reply resources/lang/en/settings.php]: As @PipoCanaja I could put the datastore name first.

Comment (SourceDoctor) [reply resources/lang/en/settings.php]: oh ok, i thought i "fixed" some of this with reordering it in section
mmh ... are subsections possible? just thinking about it
datastore name first sounds also nice :-)

Comment (murrant) [reply resources/lang/en/settings.php]: Fixed via: #11280

Comment (SourceDoctor) [reply resources/lang/en/settings.php]: you're the man ;-)

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-62-release-changelog-march-2020/11423/1

**Pull request merged**